types/trigger.d.ts

```diff
@@ -137,6 +139,7 @@ export type TriggerSet<Data> = {
   timelineTriggers?: TimelineTrigger<Data>[];
   timelineReplace?: TimelineReplacement[];
   timelineStyles?: TimelineStyle[];
+  initData?: DataInitializeFunc;
```

Comment:
```suggestion
  initData?: DataInitializeFunc<Data>;
```

Could you also convert `t1.ts` just to have a working example?